chefignore

```diff
@@ -2,7 +2,11 @@
 .kitchen
 .vagrant
 chef
+cookbooks
```

Comment:
How does this not ignore changes in `cookbooks/bcpc`, etc.? We used to have to [explicitly not ignore](https://github.com/bloomberg/chef-bach/blob/18b218eb3a5abf15d07089f860c6540cd375e4d0/.gitignore#L15-L22) them?